www_src/pages/make/make.jsx

```diff
@@ -68,6 +68,7 @@ var Make = React.createClass({
       }
       if (window.Android) {
         window.Android.setView('/projects/' + body.project.id);
+        window.Android.trackEvent('Make', 'Create a Project', 'New Project Started');
```

Comment:
This is right on. The only thing I would recommend is that we flip the order here as otherwise you have the potential for the `WebView` being dismissed before the event can be triggered.